tests/dist/threads/test_pthreads.cpp

```diff
@@ -5,6 +5,7 @@
 #include <faabric/scheduler/Scheduler.h>
 
 namespace tests {
+
```

Comment:
I usually don't add this lines myself, does `clang-format` say something about it?

Reply:
I don't see what harm this newline does, in fact, I think it makes it easier to read as it clearly separates the namespace declaration from whatever comes next. The Google style guide seems to agree as all their examples have a newline: https://google.github.io/styleguide/cppguide.html#Namespaces. 

However, doing a brief scan I can see that the repo is pretty mixed at the moment and can't find a rule in clang-tidy to enforce anything so we can leave it for now.